roster_names.md

```diff
@@ -1,3 +1,7 @@
 | Name | 
 | ---  | 
-| Mitch Phillipson | 
\ No newline at end of file
+| Mitch Phillipson | 
+
+| Name |
+| ---  |
```

Comment:
You don't need these two lines. They will make a new table. Remove these and put your name under mine